packages/core/src/types.ts

```diff
@@ -362,20 +362,25 @@ export interface ITokenDefinition {
   [key: number]: string;
 }
 
+export interface INumericTokenDefinition {
+  [key: string]: number;
+  [key: number]: number;
+}
+
 export interface ITokensDefinition {
   colors?: ITokenDefinition;
   space?: ITokenDefinition;
   fontSizes?: ITokenDefinition;
   fonts?: ITokenDefinition;
-  fontWeights?: ITokenDefinition;
-  lineHeights?: ITokenDefinition;
+  fontWeights?: INumericTokenDefinition;
```

Comment:
It would be nice to have `ITokenDefinition | INumericTokenDefinition` here

Reply:
I could not only provide `600` as a value but also `"600"` or `"bold"` as a string

Reply:
Just an idea: I think eventually it would be nice to have scale types. so for font weights it would suggest the keywords (`bold`) and numbers (`100-900`). For colors, it could suggest the color keywords (`tomato`, `rebeccapurple`).

Reply:
For `font-weight`, the numeric value can accept any numbers between 1 and 1000. My current understanding is that TypeScript will not allow union types this big.

- [Reference: microsoft/TypeScript - Suggestion: Range as Number type](https://github.com/microsoft/TypeScript/issues/15480)

Reply:
@jonathantneal Yeah, 1 to 1000 is a bit much :D From what Ive seen, 100-900 is the most common